src/transformers/modeling_outputs.py

```diff
@@ -78,6 +94,28 @@ class BaseModelOutputWithPooling(ModelOutput):
     attentions: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class BaseModelOutputWithNoAttentionAndWithPooling(ModelOutput):
```

Comment:
```suggestion
class BaseModelOutputWithPoolingAndNoAttention(ModelOutput):
```
Alternate name that would not have two with.

Reply:
okay